integration-tests/src/test/java/org/zowe/apiml/cachingservice/KeyValue.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.cachingservice;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@Data
+public class KeyValue {
```

Comment:
There is very similar static class on CachingApiIntegrationTest. Is it good idea to use just one?

Reply:
It was removed from there as it is shared by three test classes